pandas/_testing/asserters.py

```diff
@@ -440,7 +440,10 @@ def assert_is_sorted(seq) -> None:
     if isinstance(seq, (Index, Series)):
         seq = seq.values
     # sorting does not change precisions
-    assert_numpy_array_equal(seq, np.sort(np.array(seq)))
+    if isinstance(seq, np.ndarray):
+        assert_numpy_array_equal(seq, np.sort(np.array(seq)))
```

Comment:
might be faster to use libalgos.is_monotonic?

Reply:
This is used in one place, tbh I want to get rid of the method completely, but that wasn't in scope here